Update Search bar titles

dimak1 opened the pull request:
*Issue #:* bcgov/entity#11604

*Description of changes:*

- Add dynamic content above Seach criteria
- Hide Fee for MHR and Staff roles
- Updated Client Seach button conditions as per Condition Matrix
- Update/Add tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

dimak1 committed:
Add dynamic content above Seach creteria

dimak1 committed:
Update SearchBar tests

dimak1 committed:
Hide Fee for MHR of Staff roles

dimak1 commented:
Updated the text and remove the Fee (under the Search button) for certain roles.

### Client PPR and MHR view

![Client PPR MHR.png](https://images.zenhubusercontent.com/6250948d66a86efdd4f41027/a3a9640c-40de-4ede-802d-6f4cc7d8b808)

### Client PPR

![Client PPR.png](https://images.zenhubusercontent.com/6250948d66a86efdd4f41027/431d6a82-e8f9-45b4-98ad-352850e00cba)

### Staff MHR

![Staff MHR.png](https://images.zenhubusercontent.com/6250948d66a86efdd4f41027/e76df709-d1e8-4ce0-add6-b4534eb690d0)

### Staff PPR

![Staff PPR.png](https://images.zenhubusercontent.com/6250948d66a86efdd4f41027/dc86fd51-c72f-485f-b68d-5d03048e6ff2)

doug-lovett reviewed (approved):
Looks good.

cameron-eyds commented on ppr-ui/src/components/search/SearchBar.vue:
```diff
@@ -15,29 +15,16 @@
+          Select a search category and then enter a value to search. Search fees are as follows:
+          Personal Property search - $8.50; Manufactured Home search - $7.00;
+          combined Personal Property and Manufactured Home search - $12.00.
         </div>
```
Looks good but since the values are the only dynamic properties here, could we template those as computed properties vs the whole block?

dimak1 replied:
Yea, I was thinking about it, and other than saving a few lines of code I didn't see any other benefit of templating. But sure, let me refactor a bit.

dimak1 replied:
Updated.

cameron-eyds reviewed (commented):
Looks good, just the one question!

dimak1 committed:
Update text and fees above search criteria

cameron-eyds reviewed (approved):
Looks good! Thanks for the update
Good to merge when ready 👍

dimak1 committed:
Update Client Search button conditions. Add tests.

dimak1 commented:
@cameron-freshworks I've updated Client Seach button conditions as per Condition Matrix from Scott. Also added tests.

<img width="831" alt="Screen Shot 2022-05-19 at 8 09 20 AM" src="https://user-images.githubusercontent.com/2333290/169329848-75818cb7-5a4e-4270-a0a2-73dc2d333d96.png">

cameron-eyds commented:
> @cameron-freshworks I've updated Client Seach button conditions as per Condition Matrix from Scott. Also added tests.
> 
> <img alt="Screen Shot 2022-05-19 at 8 09 20 AM" width="831" src="https://user-images.githubusercontent.com/2333290/169329848-75818cb7-5a4e-4270-a0a2-73dc2d333d96.png">

@dimak1 Looks good 👍
